### PR TITLE
[SMD-348] override page widths

### PR DESCRIPTION
**Pull request by cyrusdobbs**
https://dluhcdigital.atlassian.net/browse/SMD-348

### Change description
- overrides the width of the error page to full
- overrides the width of the upload, login, success and 500 pages to two thirds

- [ ] Unit tests and other appropriate tests added or updated
- [ ] README and other documentation has been updated / added (if needed)
- [X] Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items preventing overlapping of logo")

### How to test
_If manual testing is needed, give suggested testing steps_

### CODE REVIEW

File: app/templates/main/upload.html [hunk 1]
```diff
@@ -15,68 +15,66 @@
...
+                     "label": {"text": ""},
+                     'name': 'ingest_spreadsheet'}) }}
+                {% endif %}
+                <p class="govuk-body">When you upload your return, we’ll check it for missing data and formatting errors.</p>
 
-                    {{ govukButton({
-                        "element": "button",
-                        "text": "Upload and check for errors"})
-                    }}
+            {{ govukButton({
```
Comment (Connor-OS) [inline app/templates/main/upload.html:71]: I indented this button in my previouse PR to be at the same level as the <p> above it. But it doesn't matter either way

**Review by Connor-OS — APPROVED** [review]
Looks good, tested the screens on my local

File: app/templates/main/upload.html (hunk 1 above)
Comment (cyrusdobbs) [reply app/templates/main/upload.html:71]: Good spot - I've re-indented it. 

I wonder if there is a way we can all install a pre-commit to make this consistent... I did have a brief look at one point but didn't have much luck

Comment (cyrusdobbs) [reply app/templates/main/upload.html:71]: like black for python code

**Pull request merged**